Gems/EMotionFX/Code/EMotionFX/Source/ObjectId.cpp

```diff
@@ -43,7 +43,7 @@ namespace EMotionFX
     {
         const AZ::u64 result = AZStd::stoull(text);
 
-        if (result == 0 || result == ULLONG_MAX)
+        if (result == 0 || result == AZStd::numeric_limits<unsigned long long>::max())
```

Comment:
suggestion: Maybe `uint64_t` might be nicer here? (and a bit shorter 🙂)

Reply:
The change from the ULLONG_MAX/INT_MAX_SHORT_MAX macros were done due to the `chrono.h` header no longer including limits.h.

So this code started to have compile failures due to not having access to the macro.
Instead of including the C std limits.h header, I thought it was more prudant to just update the code to use numeric_limits